docs/guide.rst

```diff
@@ -0,0 +1,191 @@
+Guide
+=====
+
+Using existing contracts
+------------------------
+
+Although it is possible to use :ref:`Client` to interact with contracts, it requires translating python values into Cairo
+values. Contract offers that and some other utilities.
+
+Let's say we have a contract with this interface:
+
+.. code-block:: python
```

Comment:
we can do just plain JSON here

Reply:
I wanted to use python value here, this way `abi` below refers to actual value